Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -13,17 +13,19 @@ however, it has to be formatted properly to pass verification tests.
 ### Changed
 
 - The artificial `ctrl`, `shift`, and `alt` controls (which combine the left and right controls into one) on the keyboard can now be written to and no longer throw `NotSupportedException` when trying to do so ([case 1340793](https://issuetracker.unity3d.com/issues/on-screen-button-errors-on-mouse-down-slash-up-when-its-control-path-is-set-to-control-keyboard)).
+- A device reset (such as when focus is lost) on `Touchscreen` will now result in all ongoing touches getting cancelled instead of all touches being simply reset to default state.
```

Comment:
Is this really both a fix and a change? Wasn't this adressed as incorrect behavior which to me would be a fix compared to a change?

Reply:
Figured that since from a user POV this affects two different APIs (`Touchscreen` vs `Touch.activeTouches`), I'd call out the change in behavior of the underlying API separately. Might be superfluous.